src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Footer.css';
+import { Link } from 'react-router-dom';
+import * as AiIcons from 'react-icons/ai';
+import cat from '../../assets/cat.png';
+
+const Footer = () => {
+  return (
+    <div className='footer-container'>
+      <div className='foot-links'>
+        <div className='footer-link-wrapper'>
+          <div class='footer-link-items'>
+            <h2>Contact Us</h2>
+              <div className='contact-info'>
+                <Link to='/'>Ivonne Hernandez</Link>
```

Comment:
We can add links to each of our names once we've figured out what we want them to link to.